Components/Link/Link.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import {Style} from '../Button/Button';
+
+export interface LinkProps {
+    active?: boolean;
+    handleClick?: (arg: any) => void;
+    className?: string;
+    disabled?: boolean;
+    href?: string;
+    style?: Style;
+    title?: string;
+    tabIndex?: number;
+    type?: string;
+    target?: string;
+}
+
+var StyledLink = styled.a`
+    display: inline;
+    padding: 0;
+    background: none;
+    border: none;
+    font-size: inherit;
+    color: #007ace;
+    text-decoration: none;
+    cursor: pointer;
+    &:hover,
+    &:active,
+    &:focus,
+    &.active {
+        color: #084e8a;
+        text-decoration: underline;
+    };
+    &.disabled {
+        cursor: default !important;
+        pointer-events: none !important;
+        color: ${props => props.style.disabledColor || 'gray'}
+    };
+`;
+
+export class Link extends React.Component<LinkProps> {
```

Comment:
We should namespace our UI components... React already has `Link`

Reply:
I would even challenge if we need to create a `<Link />` component since it's just a wrapper for an `<a />` tag.